### DISCUSSION TITLE
arm64 != RPi; different ffmpeg versions in arm64 vs amd64 containers

### DISCUSSION THREAD

**Discussion by af3556** [blakeblackshear/frigate#7197] [Ask A Question] [answered]
I am playing with frigate on two devices, one an arm64 (Jetson Nano) and the other amd64. The ffmpeg version on the arm64 image is older (4.3.6) than the amd64 (n5.1).

I see from `install_deps.sh` that the arm64 TARGETARCH is pulling from raspbian, which I presume does not have the newer ffmpeg.

I'm guessing there must be a reason why the raspbian repos are necessary for the RPi, but is there a way of refining the current TARGETARCH test to use linuxarm64 from https://github.com/BtbN/FFmpeg-Builds/releases/, and only use raspbian for the actual RPi?

**Comment by NickM-27** [comment, marked as answer]
This is already going to be done with the community supported boards work. Though it's worth mentioning in 0.12 you can already override the included ffmpeg build to use a different one.

**Reply by af3556** [reply]
Ah - indeed! I followed https://docs.frigate.video/configuration/advanced/#custom-ffmpeg-build and installed the linuxarm64 build from https://github.com/BtbN/FFmpeg-Builds/releases/tag/autobuild-2023-07-17-12-51; seems a-ok.

**Reply by snitzelweck92** [reply]
Hey. I found you post. 
And i want to update also my ffmpeg. 
Because a newer Reolink Camera makes problems. 
So i want to try it, but I'm not sure eactly how to download the package and map it for the container. 
Can you send me your docker-compose.yaml and maybe explanation in 2 sentences how you downloaded it to a specified folder?
Thanks alot :)
